buildSrc/src/main/java/io/micronaut/guides/feature/Localstack.java

```diff
@@ -26,6 +26,6 @@ public void processSelectedFeatures(FeatureContext featureContext) {
     @Override
     public void apply(GeneratorContext generatorContext) {
         super.apply(generatorContext);
-        addDependency(generatorContext, "aws-java-sdk-core", Scope.TEST);
+        addDependency(generatorContext, "aws-java-sdk-core");
```

Comment:
Yes, otherwise the ci build was failing with the compilation error:

```
Executing 'micronaut-jms-aws-sqs-maven-java' tests
Error:  COMPILATION ERROR : 
Error:  /home/runner/work/micronaut-guides/micronaut-guides/build/code/micronaut-jms-aws-sqs/micronaut-jms-aws-sqs-maven-java/src/main/java/example/micronaut/SqsClientFactory.java:[3,29] package com.amazonaws.regions does not exist
Error:  Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.10.1:compile (default-compile) on project micronautguide: Compilation failure
Error:  /home/runner/work/micronaut-guides/micronaut-guides/build/code/micronaut-jms-aws-sqs/micronaut-jms-aws-sqs-maven-java/src/main/java/example/micronaut/SqsClientFactory.java:[3,29] package com.amazonaws.regions does not exist 
```

Reply:
This should not be necessary. This will add local stack to the main classpath.

Reply:
Actually, It does not add the localstack dependency to the main classpath, it just changes the scope of the `aws-java-sdk-core `dependency. There are no issues when the demo source is generated and imported to IDE. 
However, when I revert the change, the problem occurs during the micronut-guide project build. Tests cannot be compiled because `aws-java-sdk-core` artifacts are missing.

@sdelamo what do you suggest?